ToolBox/Polygon/PolygonDataQueueHandler.cs

```diff
@@ -95,7 +97,7 @@ public PolygonDataQueueHandler(bool streamingEnabled)
         {
             if (streamingEnabled)
             {
-                foreach (var securityType in new[] { SecurityType.Equity, SecurityType.Forex, SecurityType.Crypto })
+                foreach (var securityType in new[] { SecurityType.Equity /*, SecurityType.Forex, SecurityType.Crypto */})
```

Comment:
Hi @Martin-Molinero thank you for all nice fixes. yes I forgot to put this back. I think I commented this out because at first I was subscribed to Stocks only, and this caused some warning or errors, For things to work rights one has got to be subscribed for stocks and currencies, with all these enumerations in place